newIDE/app/src/GameDashboard/Feedbacks/FeedbackCard.js

```diff
@@ -157,18 +157,28 @@ const FeedbackCard = ({
                       )}
                     </Text>
                   )}
-                  <BackgroundText style={styles.backgroundText} allowSelection>
-                    {comment.playerId ? (
-                      <Link
-                        onClick={() => setOpenPlayerPublicProfileDialog(true)}
-                        href="#"
-                      >
+                  {comment.playerId ? (
+                    <Link
+                      onClick={() => setOpenPlayerPublicProfileDialog(true)}
+                      href="#"
+                    >
+                      <Text noMargin color="inherit">
                         {comment.playerName || 'Anonymous player'}
-                      </Link>
-                    ) : (
-                      comment.playerName || 'Anonymous player'
-                    )}
-                  </BackgroundText>
+                      </Text>
+                    </Link>
+                  ) : (
+                    <BackgroundText style={styles.backgroundText}>
+                      {comment.playerName || 'Anonymous player'}
+                    </BackgroundText>
+                  )}
+                  {comment.contact && (
+                    <LineStackLayout alignItems="center" noMargin>
+                      <BackgroundText style={styles.backgroundText}>
+                        <Trans>Contact</Trans>:
```

Comment:
: might not be a valid punctuation in some language

```suggestion
                        <Trans>Contact:</Trans>
```

Reply:
good catch